src/gluonts/transform/feature.py

```diff
@@ -30,6 +30,126 @@ def target_transformation_length(
     return target.shape[-1] + (0 if is_train else pred_length)
 
 
+class MissingValueImputation:
+    """
+    The parent class for all the missing value imputation classes.
+    You can just implement your own inheriting this class.
+    """
+
+    def __init__(self) -> None:
```

Comment:
You probably need to decorate this constructor with the `@validated()` decorator to make this type compatible with the serialization/deserialization mechanism that is used. See how this is done e.g. here: https://github.com/awslabs/gluon-ts/blob/60a0d5ca1ba6d1d062aab3065b6e9c68bf45a31f/src/gluonts/transform/feature.py#L54